src/ModelSingleton.py

```diff
@@ -192,8 +192,25 @@ def __call__(self, query_text):
         output = self.model(**toks_cuda)
         cls_rep = output[0][:, 0, :]
         dist = cdist(cls_rep.cpu().detach().numpy(), self.all_reps_emb)
-        nn_index = np.argmin(dist)
-        return [self.all_reps_names[nn_index], self.all_reps_ids[nn_index]]
+        if count == 1:
+            nn_index = np.argmin(dist)
+            return [{
+                "MESH term": self.all_reps_names[nn_index],
```

Comment:
Are we always returning mesh terms , or would it be useful to generalize it to just return some structure like 
```javascript
[{ "label" :  self.all_reps_names[nn_index] , "curie" : self.all_reps_ids[nn_index], "distance_score" : round(dist[0, nn_index], 3)} ]
```